### ISSUE TITLE
FeatureRequest: Move pictures into bibnumber-folder

### ISSUE THREAD

**Issue by knutivars** [gheinrich/bibnumber#3]
When having a load of pictures from a local event, it would be awesome to point the code to a certain directory containing all the pictures, detect bib-numbers on each of the pictures (one picture might contain several), and then copy the pictures into folders representing each bib.

Example:
Folder \eventX contains:
pic1.jpg - with competitor 145
pic2.jpg - with competitor 153
pic3.jpg - with competitor 167 AND 153
pic4.jpg - none recognizable bibs.

Run the program - it will copy into folders for each numbers;
\eventX\145\ will contain pic1.jpg
\eventX\153\ will contain pic2.jpg AND pic3.jpg
\eventX\167\ will contain pic3.jpg
\eventX\NOT_FOUND\ will contain pic4.jpg

(note - pictures are copied into the folders - and pictures 3 ends up in two folders as it has two different bibs).

**Comment by gheinrich** [comment]
When you call `bibnumber` on a folder it will generate a `.csv` file with all detected bibs and for each detected bib a list of images they were found in. It should be relatively easy to write a script to then copy the image files into new folders. I am not planning to implement new features in `bibnumber` but your are welcome to contribute a Pull Request for this.

**Comment by knutivars** [comment]
Yeah, agreed - that sounds solvable. I would contribute if it were Java ;-)

**Comment by ulyrn** [comment]
I don't know about programmin, but wondering is this is already a working software and is it available, it would surely help me a lot for race photography. Thank You
